### PR TITLE
Change doc for sqaaas,

### PR DESCRIPTION
**Pull request by SergioLangaritaBenitez**
Fixes #

## Proposed Changes

 - Changes the documentation to obtain a silver badge
 - Code of conduct splitting in another file
 - metadata file created
 - changed readme, created a section of silver badge and finance
 - implementation of m42: update the documentation of endpoints
 - implementation of m59: created a funded section in the Readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ is abstracted away from the user.
 
 ### Components
 
-<img align="right" src="docs/images/oscar-components.png" alt="OSCAR Components" width="400"></left>
+![OSCAR Components](docs/images/oscar-components.png)
```
Comment (srisco) [inline README.md:60]: @SergioLangaritaBenitez Can we mantain the previous one?

File: README.md [hunk 2]
```diff
@@ -107,6 +107,29 @@ OSCAR is licensed under the Apache License, Version 2.0. See
...
+Estatal de Investigación/ 10.13039/501100011033 and by “European Union NextGenerationEU/PRTR”.
+
+Grant PID2020-113126RB-I00 funded by Ministerio de Ciencia e Innovación/Agencia
+Estatal de Investigación/ 10.13039/501100011033.
+
+<img src="docs/images/financing/logo-AEI.svg" width="120" height="120" />
+<img src="docs/images/financing/logo-gob.svg" width="120" height="120" />
+<img src="docs/images/financing/logo-ministerio-ciencia.svg" width="70" height="70" />
+<img src="docs/images/financing/logo-nextgen.png" width="150" height="50" />
+<img src="docs/images/financing/logo-PRTR2.png" width="120" height="50" />
```
Comment (srisco) [inline README.md:131]: @SergioLangaritaBenitez This should be shown side by side

File: docs/deploy-im-dashboard.md [hunk 3]
```diff
@@ -93,11 +93,25 @@ These are the steps:
...
+        where the OSCAR cluster is built.
+    * local_oscarui_endpoint: This endpoint access to OSCAR user interface. In
+        this endpoint, only the user-password process authentication is
+        allowed. It can not be accessed with EGI credentials.
+    * minio_endpoint: Endpoint where MinIO is listening to a petition. If you
+        access it by the browser, you will be redirected to
+        "console_minio_endpoint".
+    * oscarui_endpoint: Endpoint of OSCAR user interfaces that EGI or
+        user-password credentials are available. In both cases, they need the
+        endpoint of the OSCAR variable.
```
Comment (srisco) [inline docs/deploy-im-dashboard.md:113]: Review

File: README.md (hunk 2 above)
Comment (SergioLangaritaBenitez) [reply README.md]: I could put it in 1 row, but not at the same level

**Review by srisco — APPROVED** [review]

**Pull request merged**